extensions/README.md

```diff
@@ -102,6 +105,8 @@ This allows an engine to quickly determine if it supports the extensions needed
 
 To create a new extension, use the [extension template](Template.md) and open a pull request into this repo.  Make sure to add the extension to the glTF Extension Registry (top of this file).
 
+If the extension adds a new top-level array, its elements should inherit all properties of `glTFChildOfRootProperty.schema.json`. Other objects introduced by the extension should inherit all properties of `glTFProperty.schema.json`. By glTF 2.0 conventions, schemas should allow additional properties. See [`KHR_lights_punctual`](2.0/Khronos/KHR_lights_punctual) as an example.
```

Comment:
By "a new top-level array" I assume we mean an array under the root `extensions` object? Such as:

```
{
  "extensions": {
    "KHR_lights_punctual": {
      "lights": [ ... ]
    }
  }
}
```

I guess an extension could introduce a new *actual* top-level array as well, but currently we haven't done that and I'm not sure we want to. Either way we may want to specify here.

Reply:
> an array under the root `extensions` object

Yes. _Actual_ new properties could be introduced only by new spec versions.